jobsdb/setup.go

```diff
@@ -12,6 +12,10 @@ func (jd *Handle) SchemaMigrationTable() string {
 	return fmt.Sprintf("%s_schema_migrations", jd.tablePrefix)
 }
 
+func (jd *Handle) RunAlwaysSchemaMigrationTable() string {
```

Comment:
Use this [here](https://github.com/rudderlabs/rudder-server/blob/4ce36da2d4eb3c691f05c7097122c6f0e3dc0c00/jobsdb/setup.go#L37) too?